starlette/responses.py

```diff
@@ -207,9 +215,9 @@ def __init__(
         self.headers["location"] = quote(str(url), safe=":/%#?=@[]!$&'()*+,;")
 
 
-Content = typing.Union[str, bytes]
-SyncContentStream = typing.Iterable[Content]
-AsyncContentStream = typing.AsyncIterable[Content]
+_Content = typing.Union[str, bytes]
```

Comment:
Just to confirm, this is a breaking change right?

Reply:
Oh yeah, it is...